src/http/client.rs

```diff
@@ -275,7 +297,11 @@ impl Client {
                     source: Box::new(source),
                     path: path.to_string(),
                 },
-                _ => Error::Request { source }.into(),
+                _ => Error::Request {
```

Comment:
I think we can now remove this match block entirely

Reply:
Good point, cleaned that up